### PR TITLE
minor updates

### PR DESCRIPTION
**Pull request by maqi**

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @fraser999

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: src/vault.rs [hunk 1]
```diff
@@ -618,30 +631,48 @@ pub type ResponseNotifier =
...
+
+        let name = ::routing::NameType(::routing::types::vector_as_u8_64_array(
+            ::routing::types::generate_random_vec_u8(64)));
+        let value = ::routing::types::generate_random_vec_u8(1024);
+        let sign_keys =  ::sodiumoxide::crypto::sign::gen_keypair();
+        let sd = ::routing::structured_data::StructuredData::new(0, name, 0,
+            value.clone(), vec![sign_keys.0], vec![], Some(&sign_keys.1)).ok().unwrap();
+        client_routing.put_request(::routing::authority::Authority::ClientManager(client_name),
+                                   ::routing::data::Data::StructuredData(sd.clone()));
+        ::std::thread::sleep_ms(2000);
```
Comment (Fraser999) [inline src/vault.rs]: Is there a way to avoid this sleep and the subsequent one?

File: src/vault.rs [hunk 2]
```diff
@@ -435,6 +441,8 @@ impl Vault {
                 },
                 MethodCall::Reply { data } => {
                     if reply_to != None && original_data_request != None {
+                        debug!("as {:?} sending data {:?} to {:?} in responding to the ori_data_request {:?}",
+                               our_authority, data, reply_to.clone().unwrap(), original_data_request.clone().unwrap());
```
Comment (Fraser999) [inline src/vault.rs]: `unwrap` used and also in following lines.

**Comment by Fraser999** [comment]
I'd have probably made the `debug!` calls `trace!` since I can't imagine anything more verbose/low-level than these (http://doc.rust-lang.org/log/log/enum.LogLevel.html)

Also, we could maybe remove some of the `#[allow(dead_code)]` statements as we implement more of the logic.

**Pull request merged**